models/project/project.go

```diff
@@ -160,15 +263,14 @@ func (u *ProjectStruct) FetchGithub() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	var openList, closedList *list.ListStruct
+	var closedList *list.ListStruct
 	for _, list := range u.Lists() {
-		// openとcloseのリストは用意しておく
-		if list.Title.Valid && list.Title.String == config.Element("init_list").(map[interface{}]interface{})["todo"].(string) {
-			openList = list
-		} else if list.Title.Valid && list.Title.String == config.Element("init_list").(map[interface{}]interface{})["done"].(string) {
+		// closeのリストは用意しておく
+		if list.Title.Valid && list.Title.String == config.Element("init_list").(map[interface{}]interface{})["done"].(string) {
```

Comment:
__[gometalinter]__
* [gotype] invalid operation: list (variable of type *invalid type) has no field or method Title
* [gotype] invalid operation: list (variable of type *invalid type) has no field or method Title
* [gotype] undeclared name: config